pyproject.toml

```diff
@@ -57,6 +57,10 @@ Home-page = "https://github.com/scverse/anndata"
 dev = [
     # dev version generation
     "setuptools_scm",
+    # static checking
+    "black>=20.8b1",
```

Comment:
I guess people kind of get black through pre-commit? Do we need to specify it here as well?

Reply:
Thanks!